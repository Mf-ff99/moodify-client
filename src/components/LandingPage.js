import React from 'react'
import { Link } from 'react-router-dom'

export default class LandingPage extends React.Component {

    render() {
        return (
            <section className="splash-page">
                <header>
                    <h1>Welcome <span className="splash-page-to">to</span> Moodify!</h1>
                </header>
                <div className="splash-page-tutorial">
                    <p>Welcome to Moodify! Here you can track your mood and watch for particular spikes and dips. Part of being well is understanding how you feel and why you feel the way you do. Keeping track of your moods is a healthy habit to start, why not start today!</p>
                    <p>
                        Here you can enter your mood, amount of sleep, and a few sentences, or even a few paragraphs, about why you feel the way you do.</p>
                        <p>Many people find it helpful and theuraputic to record their moods as well as their thoughts. Moodify builds a comprehensive database of your moods and allows you to view, sort, and add your current mood. Please note, the more often you record your mood and the more information you record along with it, the more helpful it becomes.
      </p>
                    <Link to="/home"><button type="click" className="splash-page-btn">Start your journey!</button>
                    </Link>
                </div>
            </section>
        )
    }
}