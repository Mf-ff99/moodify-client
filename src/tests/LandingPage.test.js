import React from 'react'
import LandingPage from '../components/LandingPage'

import { shallow } from 'enzyme'

describe(`LandingPage component`, () => {
    it('Renders without crashing', () => {
        shallow(<LandingPage />)
    })
})