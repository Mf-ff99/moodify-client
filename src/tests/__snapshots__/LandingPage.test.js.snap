// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders form approriately 1`] = `
<section
  className="splash-page"
>
  <header>
    <h1>
      Welcome 
      <span
        className="splash-page-to"
      >
        to
      </span>
       Moodify!
    </h1>
  </header>
  <div
    className="splash-page-tutorial"
  >
    <p>
      Welcome to Moodify! Here you can track your mood and watch for particular spikes and dips. Part of being well is understanding how you feel and why you feel the way you do. Keeping track of your moods is a healthy habit to start, why not start today!
    </p>
    <br />
    <p>
      Here you can enter your mood, amount of sleep, and a few sentences, or even a few paragraphs, about why you feel the way you do.
    </p>
    <br />
    <p>
      Many people find it helpful and theuraputic to record their moods as well as their thoughts. Moodify builds a comprehensive database of your moods and allows you to view, sort, and add your current mood. Please note, the more often you record your mood and the more information you record along with it, the more helpful it becomes.
    </p>
    <br />
    <p>
      You can either register a new account to get started with a blank state, or log in with the pre-populated values to demo the application!
    </p>
    <Link
      to="/home"
    >
      <button
        className="splash-page-btn"
        type="click"
      >
        Start your journey!
      </button>
    </Link>
  </div>
</section>
`;
